### PR TITLE
Dev add main

### PR DESCRIPTION
**Pull request by akapush**
-Normalized package hierarchy
-Created Main class to run sorting algorithms on input
-Modified visibility of some algorithm's sort method, to make them accessible
-removed incompatible algorithms (e. g. expects a string array to sort)

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+				array.add(number);
+			} catch (NumberFormatException nfe) {
+				System.out.println("Please enter a number");
+			}
+		}
+
+		int[] res = new int[array.size()];
+		for (int i = 0; i < array.size(); i++) {
+			res[i] = array.get(i);
+		}
```
Comment (barsimark) [inline src/Main.java:160]: return array.toArray(); 
It would be a bit more readable

Comment (barsimark) [reply src/Main.java:160]: Variable name shouldn't be array as its type is ArrayList and not array

**Review by barsimark — COMMENTED** [review]
Great set of changes, only two minor corrections are needed in the Main.java

File: src/Main.java (hunk 1 above)
Comment (akapush) [reply src/Main.java:160]: Great observations, I will correct them

**Pull request merged**